src/auth.js

```diff
@@ -0,0 +1,29 @@
+const { log } = require('cozy-konnector-libs')
+const { rootUrl, request } = require('./request')
+
+const loginUrl = rootUrl + '/espace_client/connexion'
+
+module.exports = {
+  authenticate
+}
+
+// Darty temporarily disables an account after 5 login failures.
+function authenticate(login, password) {
+  log('info', 'Authenticating...')
+
+  return request({
+    method: 'POST',
+    uri: loginUrl,
+    form: {
+      email: login,
+      password: password
+    }
+  })
+    .then(() => {
+      log('info', 'Login successful.')
+    })
+    .catch(err => {
+      if (err.statusCode === 401) throw new Error('LOGIN_FAILED')
```

Comment:
You can use : 

```js
const { errors } = require('cozy-konnector-libs')
throw new Error(errors.LOGIN_FAILED)
```

FYI you can also find there the list of available error codes

Reply:
- [x] Use `errors.LOGIN_FAILED`